includes/polling/ports.inc.php

```diff
@@ -535,6 +534,17 @@
 
 
 echo "\n";
+
+// get last poll time to optimize poll_time, poll_prev and poll_period in table db
+$last_poll_time = max(array_column($ports, 'poll_time'));
+
+$device_global_ports = [];
```

Comment:
I like it a little better when arrays are defined like this:
```php
$device_global_ports = [
    'poll_time' => $polled,
    'poll_prev' => max(array_column($ports, 'poll_time')),
    'poll_period' => ($polled - $port['poll_time']),
];
```

Reply:
ok done